### ISSUE TITLE
uSync.Exporter still adds MenuRenderingNotification when disabled

### ISSUE THREAD

**Issue by LennardF1989** [KevinJump/uSync8#475]
**Latest version (v11.0.1)**

Whenever you disable uSync.Exporter with the following in appsettings:
```
"Exporter": {
  "Settings": {
    "Disabled": true
  }
},
```
The right-click menu-item for Document Types and such will still be added.

Workaround for now is adding this to a startup Composer:
```
builder.Services.Remove(
    new UniqueServiceDescriptor(
        typeof(INotificationHandler<MenuRenderingNotification>),
        typeof(ExporterMenuRenderingHandler),
        ServiceLifetime.Transient
    )
);
```

The proper fix would be that `AddNotificationHandler` is not called whenever `Disabled` is set to `true`.

Also, mild annoyance: `Disabled` is completely against your own convention where pretty much all plugins have an "Enabled" :P Would be great if the UI can actually hide disabled portions, rather than stating something is disabled.

**Comment by KevinJump** [comment]
Yes, 

That menu should be disabled when its set (fix now in for next release)

I agree the config name isn't ideal - not least because it only turns off a bit of Exporter but sounds like it should turn it all off!

To add to complexity (because we like that). I've added 

Enabled (default true)
ImportDisabled: (default: false)
ImportEnabled: (default: true).

so you can choose, but for v13, i think we will rationalize this into `ImportEnabled` 😄 

Also took the whole import box off the dashboard if it's disabled. Hopefully i don't then get the opposite, where is the "import box" issue reported, because you do have to actively turn it off. (but you never know).

**Comment by LennardF1989** [comment]
Sounds good! Thanks Kevin :)

**Comment by KevinJump** [comment]
fixed in 11.0.2
